### PR TITLE
Feature/2395 update vehicle passenger

### PR DESCRIPTION
**Pull request by FridaPZavalaHernandez**
[AB#2395](https://dev.azure.com/dsd-sdsd/ad8a8cd2-ef49-4e7c-bae2-6f8729a38334/_workitems/edit/2395)

### CODE REVIEW

File: UI/src/components/molecules/RipaStopReason.vue [hunk 1]
```diff
@@ -157,6 +157,42 @@
...
+              label="The stopped person is a passenger in a vehicle"
+            ></ripa-switch>
+
+            <v-container>
+              <v-row no-gutters>
+                <v-col cols="12" sm="12"> </v-col>
+              </v-row>
+            </v-container>
+          </template>
+          <template v-if="model.stopType === 'Pedestrian'">
```
Comment (jacobkellas) [inline UI/src/components/molecules/RipaStopReason.vue:178]: I don't know the regulation text so this is a question, if the stop is a bicycle are either of these options supposed to be available?

File: UI/tests/unit/components/molecules/RipaStopReason.spec.js [hunk 2]
```diff
@@ -90,6 +89,63 @@ describe('Ripa Stop Reason', () => {
     )
   })
 
+  it('stop type is vechiular show this question "Passenger in vehicle"', () => {
```
Comment (jacobkellas) [inline UI/tests/unit/components/molecules/RipaStopReason.spec.js:92]: typo in test name 'vehicular'

File: UI/tests/unit/components/molecules/RipaStopReason.spec.js [hunk 3]
```diff
@@ -90,6 +89,63 @@ describe('Ripa Stop Reason', () => {
...
+        },
+        statutes: computed(() => []),
+      },
+    })
+
+    expect(wrapper.html()).toContain('passenger in a vehicle')
+    expect(wrapper.html()).not.toContain('person in residence')
+  })
+
+  it('stop type is vechiular show this question "Person was inside a residence"', () => {
```
Comment (jacobkellas) [inline UI/tests/unit/components/molecules/RipaStopReason.spec.js:121]: typo in test name 'vehicular'

Comment (jacobkellas) [reply UI/tests/unit/components/molecules/RipaStopReason.spec.js:121]: And I think the name is supposed to be changed for residence

File: UI/src/components/molecules/RipaStopReason.vue [hunk 4]
```diff
@@ -157,6 +157,42 @@
             :rules="explanationRules"
             @blur="handlePiiCheck($event)"
           ></ripa-text-input>
+          <template v-if="model.stopType === 'Vehicular'">
+            <ripa-form-header
+              title="The stopped person is a passenger in a vehicle"
```
Comment (jacobkellas) [inline UI/src/components/molecules/RipaStopReason.vue:162]: Was this supposed to have the regulation section and "required" flag?

File: UI/src/components/molecules/RipaStopReason.vue [hunk 5]
```diff
@@ -157,6 +157,42 @@
...
+
+            <v-container>
+              <v-row no-gutters>
+                <v-col cols="12" sm="12"> </v-col>
+              </v-row>
+            </v-container>
+          </template>
+          <template v-if="model.stopType === 'Pedestrian'">
+            <ripa-form-header
+              title="The stopped person was inside a residence..."
```
Comment (jacobkellas) [inline UI/src/components/molecules/RipaStopReason.vue:180]: Same question here about required flag and regulation section

**Review by jacobkellas — CHANGES_REQUESTED** [review]

File: UI/src/components/molecules/RipaStopReason.vue (hunk 1 above)
Comment (FridaPZavalaHernandez) [reply UI/src/components/molecules/RipaStopReason.vue:180]: In my understanding the questions should not be available for a stop type bicycle. Only Vehicular or Pedestrian.

File: UI/src/components/molecules/RipaStopReason.vue [hunk 6]
```diff
@@ -157,6 +157,48 @@
             :rules="explanationRules"
             @blur="handlePiiCheck($event)"
           ></ripa-text-input>
+          <template v-if="model.stopType === 'Vehicular'">
```
Comment (jacobkellas) [inline UI/src/components/molecules/RipaStopReason.vue:160]: Can you add a "&& model.stopVersion === 2" just to make sure it doesn't render for V1 stops?

File: UI/src/components/molecules/RipaStopReason.vue [hunk 7]
```diff
@@ -157,6 +157,48 @@
...
+              label="The stopped person is a passenger in a vehicle"
+            ></ripa-switch>
+
+            <v-container>
+              <v-row no-gutters>
+                <v-col cols="12" sm="12"> </v-col>
+              </v-row>
+            </v-container>
+          </template>
+          <template v-if="model.stopType === 'Pedestrian'">
```
Comment (jacobkellas) [inline UI/src/components/molecules/RipaStopReason.vue:181]: Same thing here

**Review by jacobkellas — CHANGES_REQUESTED** [review]

**Review by jacobkellas — APPROVED** [review]

**Pull request merged**